paymentsheet/src/main/java/com/stripe/android/paymentsheet/verticalmode/PaymentMethodVerticalLayoutUI.kt

```diff
@@ -176,13 +177,21 @@ private fun ViewMoreButton(
     Row(
         verticalAlignment = Alignment.CenterVertically,
         modifier = Modifier
+            .focusable()
             .testTag(TEST_TAG_VIEW_MORE)
-            .clickable(onClick = onViewMorePaymentMethods)
+            .clickable {
+                onViewMorePaymentMethods()
+            }
             .padding(4.dp)
             .fillMaxHeight()
     ) {
         Text(
-            stringResource(id = R.string.stripe_view_more),
+            modifier = Modifier
+                .clickable {
```

Comment:
Why are we adding a second click listener here? The one on the row should be enough.